content/influxdb/cloud/get-started/setup.md

```diff
@@ -28,11 +28,11 @@ you need is in place.
     command line. For detailed installation and setup instructions,
     see [Use the influx CLI](/influxdb/cloud/tools/influx-cli/).
 
-2.  **Create an All Access API token.**
+2.  **Create an All-Access API token.**
```

Comment:
> I agree that this is what it _should_ be, but it doesn't include the hyphen in the UI:
> 
> <img alt="image" width="254" src="https://private-user-images.githubusercontent.com/181484/311876061-fc29e3cd-33f8-40ad-87a3-614e28c4623d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyNTExMzUsIm5iZiI6MTcxMDI1MDgzNSwicGF0aCI6Ii8xODE0ODQvMzExODc2MDYxLWZjMjllM2NkLTMzZjgtNDBhZC04N2EzLTYxNGUyOGM0NjIzZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxMlQxMzQwMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MTE4YzFkZDEyYzJjZTYwZTJkYzUxMGYwYzJjY2RiMjUwOTdhYWNiNDFiNTk5Nzc1NDZjNDM1YTExMjFmODczJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.do7qlAhj8bJ6Self4F-De3682q93NADPZkvd2dufg9w">
> I think we should either leave the docs the same, or update it in the UI: https://github.com/influxdata/ui/blob/167c4c65217ef2f6d1d91e0b5facafe5590ac8c4/src/authorizations/components/GenerateTokenDropdown.tsx#L27

Thanks. My bad, I intended to ignore the UI samples, but got too trigger happy replacing. I mostly just want it to be consistent.

Reply:
100% agree on consistency. Honestly, I'm all for updating the UI. It's a simple change. The only problem is that I don't know when it will make it into OSS. Whenever the next release is I guess. But it would make it into Cloud a few hours after it's merged.